hardhat.config.ts

```diff
@@ -15,6 +16,182 @@ const accounts: HDAccountsUserConfig = {
     'test test test test test test test test test test test junk',
 }
 
+const SUPPORTED_NETWORKS = [
+  'celo',
+  'mainnet',
+  'arbitrum',
+  'polygon',
+  'op',
+  'base',
+  'berachain',
+  'vana',
+]
+
+const ONE_DAY = 60 * 60 * 24
+
+task('deploy-registry', 'Deploy Registry contract')
```

Comment:
supernit: wondering if we should use this convention? for more visual grouping of tasks.
The hardhat task guide mentions this for subtasks: https://hardhat.org/hardhat-runner/docs/advanced/create-task#subtasks
But well I guess here it's not really a subtask. Anyway doesn't matter either way.

```suggestion
task('deploy:registry', 'Deploy Registry contract')
```